opennlp-tools/src/main/java/opennlp/tools/util/featuregen/BrownCluster.java

```diff
@@ -36,34 +36,40 @@
 /**
  *
  * Class to load a Brown cluster document: word\tword_class\tprob
- * http://metaoptimize.com/projects/wordreprs/
- *
+ * <p>
+ * See: <a href="http://metaoptimize.com/projects/wordreprs/">
```

Comment:
Link seems dead (at least for me). Getting a "Origin is unreachable Error code 523" from cloudflare.
It isn't archived in the InternetArchive either. I was only able to find a copy via `archive.ph`: https://archive.ph/p82oc

I think the related research paper is: https://dl.acm.org/doi/10.5555/1858681.1858721

Maybe we can add a reference and mark the link as "broken" ?